lib/index.js

```diff
@@ -97,10 +70,13 @@ module.exports = class Breakpoints {
     const userConfigPath = path.resolve(process.cwd(), this.#userConfigName);
     try {
       this.#userConfig = deepmerge(
-        DEFAULT_CONFIG,
+        this.#userConfig,
         Breakpoints.readYamlFile(userConfigPath)
       );
-      if (!this.#userConfig.extractTo?.css && !this.#userConfig.extractTo?.js) {
+      if (!this.#userConfig.css?.enabled && !this.#userConfig.js?.enabled) {
+        console.warn(MESSAGES.noOutput(this.#userConfig));
+      }
+      if (Object.values(this.#userConfig.options).every((option) => option)) {
```

Comment:
Took me a sec to understand this. Maybe add a comment to explain what’s happening or rewrite the code to be more clear?

Maybe something like this:

```js
if (!Object.values(this.#userConfig.options).includes(true)) { …
```